daemon/algod/api/server/v2/handlers.go

```diff
@@ -651,6 +651,35 @@ func (v2 *Handlers) GetBlock(ctx echo.Context, round uint64, params model.GetBlo
 	return ctx.Blob(http.StatusOK, contentType, data)
 }
 
+// GetBlockTxids gets all top level TxIDs in a block for the given round.
+// (GET /v2/blocks/{round}/txids)
+func (v2 *Handlers) GetBlockTxids(ctx echo.Context, round uint64) error {
+	ledger := v2.Node.LedgerForAPI()
+	block, _, err := ledger.BlockCert(basics.Round(round))
```

Comment:
use `ledger.Block`, not `ledger.BlockCert` - the former is lighter from db lookup point of view. Btw, looks like all `BlockCert` usages can be replaced with `Block`.

Reply:
Good point, I changed this and the other instances in [833fae2](https://github.com/algorand/go-algorand/pull/5611/commits/833fae2d46bb7f5a1c22d1fe92c53264e51bc971)